src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1750,6 +1750,13 @@ private CSDiagnosticInfo NotFound(CSharpSyntaxNode where, string simpleName, int
             bool encounteredForwardingCycle;
             string fullName;
 
+            // for attributes, suggest both, but not for verbatim name
+            if (options == LookupOptions.AttributeTypeOnly)
```

Comment:
I understand, but checking for the flag would make both attribute and verbatim name attribute valid. From testing, I could not see that attribute type options are ever combined with others. I will add a Debug.Assert to check.